build/istio/istioctl-values.yaml

```diff
@@ -34,8 +34,6 @@ spec:
           minReplicas: 1
           maxReplicas: 1
   values:
-    prometheus:
-      enabled: false
```

Comment:
Note the default is `true` here. This may conflict with the official cf-k8s-prometheus project

Reply:
Nope, I wasn't aware. Thanks!